packages/beacon-node/test/spec/presets/index.test.ts

```diff
@@ -22,7 +22,16 @@ import {transition} from "./transition.js";
 const skipOpts: SkipOpts = {
   skippedForks: [],
   skippedRunners: [],
-  skippedHandlers: ["full_withdrawals", "partial_withdrawals", "bls_to_execution_change", "withdrawals"],
+  skippedHandlers: [
+    "full_withdrawals",
+    "partial_withdrawals",
+    "bls_to_execution_change",
+    "withdrawals",
+    "historical_summaries_update",
```

Comment:
added to pending tracker list 
-https://github.com/ChainSafe/lodestar/issues/4680, 
but other transition tests cover summaries update as well  so we can take it up as a separate task

Reply:
added the epochfn handler for historical_summaries_update, so its not skipped now :+1: